securedrop/source.py

```diff
@@ -336,16 +336,25 @@ def login():
             flash("Sorry, that is not a recognized codename.", "error")
     return render_template('login.html')
 
+
 @app.route('/logout')
 def logout():
     if logged_in():
         session.clear()
-        flash("Thank you for logging out.", "notification")
-
+        tor_msg = Markup("""<strong>Important:</strong><br>
+                         Thank you for logging out!<br>
+                         Please fully end your session by restarting
+                         Tor Browser:<br>
+                         1. Click the
+                           <img src="static/i/toronion.png" alt="Tor icon" />
+                           Tor onion icon in the toolbar above.<br>
+                         2. Click <strong>  New Identity</strong>.<br>
+                         3. Click <strong>Yes</strong> in the dialog box
+                           that appears.""")
```

Comment:
@garrettr Good point - modified this in 2905b42 and added the new template to the AppArmor rules in 5fe7d40